### PR TITLE
unit test/cmd

### LINKED ISSUE #227: Validator Unit Tests
- [x] Helpers
- [x] CMD Clients
- [x] CMD Repositories
- [x] CMD Services
- [x] Model - auth-messages
- [x] Model - burn-event
- [x] Model - messages
- [x] Model - transfer
- [ ] Core - Pair
- [ ] Core - Queue
- [ ] Core - Server
- [ ] Clients - Ethereum
- [ ] Clients - Hedera Node
- [ ] Clients - Mirror Node
- [x] Burn event Service
- [ ] Scheduling Service
- [ ] Contracts service
- [ ] Messages service
- [ ] Signer service
- [ ] Transfers service
- [ ] Persistence 
- [ ] Handlers
- [ ] Watchers
- [ ] Recovery
- [ ] Router

### PR DESCRIPTION
**Pull request by Holymir**
**Detailed description**:
cmd unit tests

**Which issue(s) this PR fixes**:
Fixes 227 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: cmd/clients_test.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import (
+	tc "github.com/limechain/hedera-eth-bridge-validator/test/test-config"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestPrepareClients(t *testing.T) {
+	res := PrepareClients(tc.TestConfig.Validator.Clients)
+	assert.NotEmpty(t, res)
```
Comment (georgiyazovaliiski) [inline cmd/clients_test.go:27]: Maybe we should be asserting that the proper clients are prepared. Even if they are just mocked(or empty).

File: cmd/repositories_test.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	"github.com/stretchr/testify/assert"
+	"gorm.io/gorm"
+	"testing"
+)
+
+func TestPrepareRepositories(t *testing.T) {
+	mocks.Setup()
+	mocks.MDatabase.On("GetConnection").Return(&gorm.DB{})
+	res := PrepareRepositories(mocks.MDatabase)
+	assert.NotEmpty(t, res)
```
Comment (georgiyazovaliiski) [inline cmd/repositories_test.go:30]: Maybe we should be asserting that the proper repositories are prepared. Even if they are just mocked(or empty).

File: test/test-config/test-config.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test_config
+
+import "github.com/limechain/hedera-eth-bridge-validator/config"
+
+var (
+	TestConfig = config.Config{
```
Comment (georgiyazovaliiski) [inline test/test-config/test-config.go:22]: Still not sure whether we have to instantiate `clients/repositories/services` with a real configuration. `PrivateKey:            "9f6da11eecc0fd7cb081d2aee88092ee3436397916c894ad6cd80a79009c0ded"` - is this safe?

**Review by georgiyazovaliiski — CHANGES_REQUESTED** [review]

File: test/test-config/test-config.go (hunk 3 above)
Comment (Holymir) [reply test/test-config/test-config.go:22]: Those are random generated pr keys

**Review by georgiyazovaliiski — APPROVED** [review]

**Pull request merged**